From: Thomas Bogendoerfer <tbogendoerfer@suse.de>
Subject: RDMA/mana: hide new rdma_driver_ids
Patch-mainline: Never, kABI workaround
References: bsc#1210741 jsc#PED-4022

Upstream commit 0266a177631d adds new enums to rdma_driver_id, which
breaks kABI. This patch hides them from the checker.

Signed-off-by: homas Bogendoerfer <tbogendoerfer@suse.de>
---
 include/uapi/rdma/ib_user_ioctl_verbs.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/uapi/rdma/ib_user_ioctl_verbs.h
+++ b/include/uapi/rdma/ib_user_ioctl_verbs.h
@@ -250,8 +250,10 @@ enum rdma_driver_id {
 	RDMA_DRIVER_QIB,
 	RDMA_DRIVER_EFA,
 	RDMA_DRIVER_SIW,
+#ifndef __GENKSYMS__
 	RDMA_DRIVER_ERDMA,
 	RDMA_DRIVER_MANA,
+#endif
 };
 
 enum ib_uverbs_gid_type {
